### ISSUE TITLE
Cooldown On showing Items

### ISSUE THREAD

**Issue by WalkAway7** [SmashMC-Development/Suggestions#843]
**Describe the suggestion**
Before the reset, there was a cooldown for how long someone could show a pokemon. This was to deal with people who would spam it. I've recently seen that people spam show Items and pokemon, with the newer command. I would like to have a cooldown again, maybe on of 5 mins.

**Which servers does this apply to?**
All pixelmon servers

**Additional context**
N/A

**Comment by HobnobberJunior** [comment]
I think 5 mins might be a little too long in my opinion. However I do think a cooldown is a good idea.

**Comment by ghost** [comment]
I like the new plugin and think it doesn't need cooldown. A player can spam anything if they really want to and feel like breaking the rule. So, doesn't make sense put a cooldown. If they want to spam, they will just spam something else.

**Comment by HobnobberJunior** [comment]
It is still one thing less a player can spam, so I would see that as a win, but I do get where you are coming from.

**Comment by WalkAway7** [comment]
I suggested this to the team before putting it up, and I have been thinking that so long as people keep them to a single message, I don't see what could be wrong. But when people spam the chats with tonnes of megastones and pokemon, it gets kinda annoying/distracting. I think if people where to use the showing items/pokemon in a way that was more Conservative then I wouldn't have to much of a problem, but the thing is, is that people spam the chat with megastones or pokemon, ask if someone is buying, then do it again like 1 min later

**Comment by HobnobberJunior** [comment]
I totally agree with this and I think this would solve this problem of people constantly showing the same mons that they want to sell. I just think that the timer shouldn't be too long. I remember that before reset it felt like ages before you we're able to show another mon. I think 5 mins if a bit too long, but I might be wrong.

**Comment by Mikirae** [comment]
Looking into this currently but I'm not sure if there's an option in the plugin itself, so we'll probably need to get something made.

**Comment by ghost** [comment]
If cooldown is something like rtp cooldown then it could work. But the cooldown on /show was too big before reset.

**Comment by Mikirae** [comment]
So RTP belongs to Nucleus which has a person configuration for cooldowns,
as do many other things. The only config for the show plugin that I'm
seeing has nothing of value in it whatsoever. I'm pretty sure there's a
custom cooldown plugin that we have somewhere and I'm looking for it right
now.

**Comment by Mikirae** [comment]
Nevermind, that would only work with commands and this is just a chat
interaction. Yeah, going to need to get something custom for this
unfortunately.

**Comment by ghost** [comment]
> If cooldown is something like rtp cooldown then it could work. But the cooldown on /show was too big before reset.

I meant in terms of duration. Like 5-10 mins would be too long.

**Comment by ghost** [comment]
Also, thanks for the update :)

**Comment by Mikirae** [comment]
Ah, I unfortunately don't really remember what RTP cooldown is. For show, I
was probably going to do it between 1-3 minutes, more than likely settling
at 2 minutes.

**Comment by Mikirae** [comment]
No problem. Thanks for the suggestions <3

**Comment by ghost** [comment]
> Ah, I unfortunately don't really remember what RTP cooldown is. For show, I was probably going to do it between 1-3 minutes, more than likely settling at 2 minutes.

Yes, I think that would be more than sufficient to stop the spamming.
